README.md

```diff
@@ -28,9 +28,12 @@ After the bootstrap script has completed, the regular gradlew instructions below
 
 #### Scala and YARN
 
-Samza builds with [Scala](http://www.scala-lang.org/) 2.11 or 2.12 and [YARN](http://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html) 2.6.1, by default. Use the -PscalaSuffix switches to change Scala versions. Samza supports building Scala with 2.11 and 2.12.
+Samza builds with [Scala](http://www.scala-lang.org/) 2.11 or 2.12 and [YARN](http://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html) 2.10.1 or 3.3.4. Scala 2.12 and Yarn 2.10.1 are used by default. Use the -PscalaSuffix and -PyarnVersion switches to change Scala or Yarn versions. Samza supports building Scala with 2.11 and 2.12.
```

Comment:
I think that we should still keep the original description, w/ default to 2.10.1. The separate build commands below are also not needed.

We may want to add a section separately regarding to JDK11 runtime support, i.e. stating that Samza currently is still built w/ Java8 but support Java11 runtime, w/ samza-yarn3 built on YARN 3.3.4.

Reply:
Also, please list the modules that support jdk11 runtime in the above section (i.e. not samza-yarn, samza-shell, samza-test, and samza-hdfs, since those modules are still on YARN 2.10.1).